PostFreeSurfer/scripts/CreateMyelinMaps_1res.sh

```diff
@@ -146,16 +141,17 @@ for LowResMesh in ${LowResMeshes} ; do
 done
 
 #Add CIFTI Maps to Spec Files
+
+STRINGIIList="corrThickness@dscalar"
+if [ "${T2wPresent}" = "YES" ] ; then
+  STRINGIIList+=" MyelinMap_BC@dscalar SmoothedMyelinMap_BC@dscalar"
+fi
+  
 for STRING in ${STRINGII} ; do
```

Comment:
Looks like this should be `STRINGIIList`.

Reply:
Never mind, I didn't look at the context.  I would argue that we shouldn't use `STRINGII` to store a list and then loop over it, as it looks wrong.

Reply:
However, that is our fault, not yours.

Reply:
I agree, and I had already previously flagged this exact confusion as item 6 in #146 for some post PR cleanup that is our responsibility.